Dockerfile

```diff
@@ -34,32 +34,35 @@ RUN { \
 
 ###
 
-FROM alpine:3.19.1 as base-cron
+FROM alpine:3.19.1 as cron
 
 ARG user=crooner
 RUN addgroup --gid 3001 ${user} && adduser -D -G ${user} -g "${user} user" -u 3001 ${user}
 
-RUN apk add dpkg curl
```

Comment:
I tried using `wget` I couldn't stop the request from downloading the response. 
Full disclosure; I didn't test very much with it, it failed and I used curl 😄 

Happy to give it an extra 30 minutes to save on potential tech debt though, and of course, squeeze some weight out of the image 💪

Reply:
This might be one way of doing it https://superuser.com/a/197043/1097788

But... one thing I didn't consider... curl is more popular and devs will be more familiar with it. 

Maybe using an unfamiliar tool is also tech debt 😅